packages/common/src/entities/dbmss/dbmss.local.ts

```diff
@@ -469,12 +478,30 @@ export class LocalDbmss extends DbmssAbstract<LocalEnvironment> {
         const dbmsIdFilename = `dbms-${dbmsId}`;
 
         if (await fse.pathExists(path.join(dbmssDir, dbmsIdFilename))) {
-            throw new DbmsExistsError(`DBMS with name ${dbmsIdFilename} already exists`);
+            throw new DbmsExistsError(`DBMS: ${dbmsIdFilename} already exists`);
         }
 
-        await fse.copy(extractedDistPath, path.join(dbmssDir, dbmsIdFilename));
+        let targetDirAlreadyExists = false;
 
         try {
+            if (targetDir) {
+                targetDirAlreadyExists = await fse.pathExists(targetDir);
+                if (!targetDirAlreadyExists) {
+                    await fse.ensureDir(targetDir);
+                }
+                const files = (await fse.readdir(targetDir)).filter((file) => !file.startsWith('.'));
```

Comment:
Hidden files are still files and we should fail in that case too because the directory is not empty.

```suggestion
                const files = await fse.readdir(targetDir);
```

Reply:
Yeah I understand, I can remove this. It may get annoying with things like `.DS_Store` on Mac for example. At the same time, don't want to start listing all the files in the directory in the error message as there could be too many. So for now, makes sense to remove it.